cmd/describe/volume.go

```diff
@@ -122,8 +146,31 @@ func (c *volumeCommand) runWithCtx(ctx context.Context, cmd *cobra.Command, args
 			return err
 		}
 
+		// filter by labels
 		volumes = set.FilterVolumes(volumes)
 
+		// filter by replica count
+		if c.replicaCountFilter != -1 {
+			volumes = selectors.FilterVolumesByReplicaCount(volumes, c.replicaCountFilter)
+		}
+
+		// filter by master health state
+		// the invalid option is used when there's no intention of filtering (no user input)
+		if c.masterHealthState != health.Invalid {
+			volumes = selectors.FilterVolumesByMasterState(volumes, c.masterHealthState)
+		}
+
+		// filter by replicas health state
+		// the invalid option is used when there's no intention of filtering (no user input)
+		if c.replicasHealthState != health.Invalid {
```

Comment:
Same point about having an explicit `replicasHealthStateFilterEnabled` and `masterHealthStateFilterEnabled` rather than encoding it with `health.Invalid`

Reply:
Added